src/Calculator/EquationStylePanelControl.xaml.cpp

```diff
@@ -79,3 +90,149 @@ void EquationStylePanelControl::SelectColor(Color selectedColor)
         }
     }
 }
+
+void EquationStylePanelControl::OnSelectedStylePropertyChanged(EquationLineStyle, EquationLineStyle newStyle)
+{
+    SelectStyle(newStyle);
```

Comment:
Can you add TraceLogger::GetInstance()->LogGraphLineStyleChanged(LineStyleType::Pattern); You'll need to update the LineStyleType enum (you can pick a different name if you don't like Pattern).